assets/app.js

```diff
@@ -1,2 +1,6 @@
 // All things loaded by this entrypoint is available everywhere
-import './scss/main.scss';
\ No newline at end of file
+import './js/vendor/theme.js';
```

Comment:
Codacy has a fix for the issue: [Replace `'./js/vendor/theme.js'` with `"./js/vendor/theme.js"`](https://app.codacy.com/gh/acecconato/AnthonyCecconato_5_18102021/pullRequest?prid=9466078)
```suggestion
import "./js/vendor/theme.js";
```